src/applications/healthcare/questionnaire/app-entry.jsx

```diff
@@ -0,0 +1,14 @@
+import 'platform/polyfills';
+import './sass/questionnaire.scss';
+
+import startApp from 'platform/startup';
+
+import routes from './routes';
+import reducer from './reducers';
+import manifest from './manifest.json';
+
+startApp({
+  url: manifest.rootUrl,
+  reducer,
+  routes,
+});
```

Comment:
If you're going to have analytics at some point and want your app to load with access to `document.dataLayer` you need to include a key/value like this: `analyticsEvents: []`

Reply:
Good to know!